codegen/ui_tests/rhai_fn_rename_collision_with_itself.stderr

```diff
@@ -0,0 +1,17 @@
+error: duplicate Rhai signature for 'foo'
+  --> $DIR/rhai_fn_rename_collision_with_itself.rs:12:15
+   |
+12 |     #[rhai_fn(name = "foo", name = "bar", name = "foo")]
+   |               ^^^^
+
+error: duplicated function renamed 'foo'
+  --> $DIR/rhai_fn_rename_collision_with_itself.rs:12:15
+   |
+12 |     #[rhai_fn(name = "foo", name = "bar", name = "foo")]
+   |               ^^^^
```

Comment:
This error message isn't great. But since I have a feeling it's difficult to fix, and everything is on one line, I will accept it for now.